settingshelper.py

```diff
@@ -199,3 +200,37 @@ def fix_logger_obfuscation(fix_logger_obfuscation_, logging_config):
                         'logging.StreamHandler'
                     ), file=sys.stderr)
                 handler["class"] = "logging.StreamHandler"
+
+
+def configure_sentry(base_dir, server_env, pub_key, priv_key, project_id):
+    if not (pub_key and priv_key and project_id):
+        return
+
+    try:
+        from raven import breadcrumbs, fetch_git_sha
+    except ImportError:
+        return
+
+    release_dir = base_dir.split('/')[-1]
+    if re.match('\d{4}-\d{2}-\d{2}_\d{2}.\d{2}', release_dir):
+        release = "{}_{}".format(server_env, release_dir)
```

Comment:
It appears in the error tags:

![image](https://cloud.githubusercontent.com/assets/249606/23022375/1d36d582-f459-11e6-8b8e-c13912817d65.png)

And on the releases page:

![image](https://cloud.githubusercontent.com/assets/249606/23022386/2df6fb9a-f459-11e6-91a2-501e50d83f40.png)

Making it the URL to the tag would be long (https://github.com/dimagi/commcare-hq/releases/tag/2017-02-16_09.43-staging-deploy) but I think it'll change the format to match the tag name.